transformer/types/index.d.ts

```diff
@@ -0,0 +1,50 @@
+/// <reference types="assemblyscript/std/assembly/index" />
+
+declare function fileToByteArray(filePath: string): StaticArray<u8>;
```

Comment:
Don't you think that we document all that ?

Reply:
I added some stuffs about that in the readme